pkg/server/conn.go

```diff
@@ -1724,6 +1732,61 @@ func (cc *clientConn) audit(eventType plugin.GeneralEvent) {
 	}
 }
 
+// handleQueryAttributes support query attributes since mysql 8.0.23
+// see https://dev.mysql.com/doc/refman/8.0/en/query-attributes.html
+// https://archive.fosdem.org/2021/schedule/event/mysql_protocl/attachments/slides/4274/export/events/attachments/mysql_protocl/slides/4274/FOSDEM21_MySQL_Protocols_Query_Attributes.pdf
+func (cc *clientConn) handleQueryAttributes(ctx context.Context, data []byte) (pos int, err error) {
```

Comment:
Maybe `parseQueryAttributes` is a better name. `handleXXX` is usually used when `XXX` is a command (e.g. `handleStmtSendLongData`...).